bitnami/pytorch/templates/deployment.yaml

```diff
@@ -0,0 +1,143 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "pytorch.fullname" . }}-master
+  labels:
+    app.kubernetes.io/name: {{ include "pytorch.name" . }}
+    helm.sh/chart: {{ include "pytorch.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/component: "master"
+spec:
+  replicas: 1 
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "pytorch.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      app.kubernetes.io/component: "master"
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "pytorch.name" . }}
+        helm.sh/chart: {{ include "pytorch.chart" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/managed-by: {{ .Release.Service }}
+        app.kubernetes.io/component: "master"
+    spec:
+      {{- include "pytorch.imagePullSecrets" . | nindent 6 }}
+      {{- if .Values.securityContext.enabled }}
+      securityContext:
+        fsGroup: {{ .Values.securityContext.fsGroup }}
+        runAsUser: {{ .Values.securityContext.runAsUser }}
+      {{- end }}
+      {{- if .Values.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.nodeSelector | indent 8 }}
```

Comment:
You can use a single line using `nindent`:

```yaml
      nodeSelector: {{ toYaml .Values.nodeSelector | nindent 8 }}
```

Apply it to affinity, tolerations, etc.